### PR TITLE
LG-43765 - update to Studio 2024

### PR DESCRIPTION
**Pull request by cornea-robert**

### CODE REVIEW

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/Sdl.SDK.LanguagePlatform.Samples.TmAutomation.csproj [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\..\Program Files %28x86%29\Trados\Trados Studio\Studio17\</OutputPath>
+    <OutputPath>..\..\..\..\Program Files %28x86%29\Trados\Trados Studio\Studio18\</OutputPath>
```
Comment (Impertatore) [inline LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmAutomation/Sdl.SDK.LanguagePlatform.Samples.TmAutomation.csproj:19]: Change to: $(MSBuildProgramFiles32)?

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmLookup/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Sdl.SDK.LanguagePlatform.Samples.TmLookup")]
-[assembly: AssemblyCopyright("Copyright ©  2023")]
+[assembly: AssemblyCopyright("Copyright ©  2024")]
```
Comment (Impertatore) [inline LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmLookup/Properties/AssemblyInfo.cs:12]: Reminder: We'll need to review the copyright for all of these in a separate PR

Example:
[assembly: AssemblyCompany("SDL Limited as part of the RWS Holdings Plc group of companies")]
[assembly: AssemblyCopyright("Copyright © 2011 - 2022 SDL Limited as part of the RWS Holdings Plc group of companies (\"RWS Group\").")]

File: ProjectAutomation/ProjectCreationSample/Sdl.ProjectAutomation.ProjectCreationSample.Logic/Sdl.ProjectAutomation.ProjectCreationSample.Logic.csproj [hunk 3]
```diff
@@ -16,7 +16,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio17\</OutputPath>
+    <OutputPath>$(ProgramFiles)\SDL\SDL Trados Studio\Studio18\</OutputPath>
```
Comment (Impertatore) [inline ProjectAutomation/ProjectCreationSample/Sdl.ProjectAutomation.ProjectCreationSample.Logic/Sdl.ProjectAutomation.ProjectCreationSample.Logic.csproj:19]: should this be: $(MSBuildProgramFiles32)?  also in other project files?

Comment (cornea-robert) [reply ProjectAutomation/ProjectCreationSample/Sdl.ProjectAutomation.ProjectCreationSample.Logic/Sdl.ProjectAutomation.ProjectCreationSample.Logic.csproj]: done

File: LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmLookup/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (cornea-robert) [reply LanguagePlatform/Sdl.SDK.LanguagePlatform.Samples.TmLookup/Properties/AssemblyInfo.cs]: Done

**Review by Impertatore — APPROVED** [review]

**Pull request merged**